config/manager/manager.yaml

```diff
@@ -89,3 +89,23 @@ spec:
   selector:
     matchLabels:
       control-plane: controller-manager
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: operator-ingress
+  namespace: k4e-operator-system
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /
+spec:
+  rules:
+    - host: operator.info
```

Comment:
This is correct, or should be a parameter somewhere?

Reply:
I'd exclude this section from this file, else its output will be used to generate the operator yaml which isn't desired for all cases.
As Eloy suggested, for dev-env, user can provide the hostname as parameter to be used or fallback to a default

Reply:
done